pkgs/top-level/all-packages.nix

```diff
@@ -15685,6 +15687,9 @@ in
   nettools = if stdenv.isLinux then callPackage ../os-specific/linux/net-tools { }
              else unixtools.nettools;
 
+  nettools_mptcp = if stdenv.isLinux then callPackage ../os-specific/linux/net-tools/mptcp.nix { }
+             else unixtools.nettools;
```

Comment:
This makes little sense as you would rather see nettols_mptcp fail than installing unixtools.nettools without warnings. This is a good fallback for `nettols` but not for `nettols_mptcp`

Reply:
I've removed the if.